.github/workflows/publish.yml

```
@@ -42,3 +45,40 @@ jobs:
 
       - name: Deploy To PyPi
         run: poetry publish -u ${{ secrets.PYPI_USERNAME }} -p ${{ secrets.PYPI_TOKEN }}
+
+  ghcr:
+    runs-on: ubuntu-20.04
+    permissions:
+      contents: read
+      packages: write
+    if: github.repository_owner == 'Virtool'
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Write VERSION file
+      run: echo ${{ github.event.release.tag_name }} > VERSION
+
+    - name: Update pyproject.toml version
+      run: sed -i 's/0\.0\.0/${{ github.event.release.tag_name }}/' pyproject.toml
+
+    - name: Login to Registry
+      uses: docker/login-action@v1
+      with:
+        registry: ${{ env.REGISTRY }}
+        username: ${{ secrets.GH_USERNAME }}
+        password: ${{ secrets.GH_TOKEN }}
+
+    - name: Extract Metadata
+      id: meta
+      uses: docker/metadata-action@98669ae865ea3cffbcbaa878cf57c20bbf1c6c38
+      with:
+        images: ${{ env.REGISTRY }}/virtool/virtool-workflow
```

To match Dockerhub tagging.
```suggestion
        images: ${{ env.REGISTRY }}/virtool/workflow
```